packages/shared/src/svg/BetaBadge.tsx

```diff
@@ -5,18 +5,16 @@ export default function BetaBadge({
 }: HTMLAttributes<SVGElement>): ReactElement {
```

Comment:
Not particular to this PR, but this looks to be something in the cloudinary.

Reply:
@sshanzel not sure what you mean here?

Reply:
I mean this asset I think should have been in the cloudinary when it was introduced in the past. I could be wrong though.

Reply:
Since it was already present I'll leave it for now. 🙏

Reply:
Yep. Agreed with that.